src/routes/volume/CreateVolume.js

```diff
@@ -255,6 +233,43 @@ const modal = ({
             </Select>)}
           </FormItem>
         </Spin>
+        <Collapse>
+          <Panel header="Advanced Configurations" key="1">
+            <FormItem label="Update Snapshot Data Integrity" hasFeedback {...formItemLayoutForAdvanced}>
+              {getFieldDecorator('snapshotDataIntegrity', {
+                initialValue: defaultSnapshotDataIntegrityValue,
+              })(<Select>
+              { defaultSnapshotDataIntegrityOption.map(option => <Option key={option.key} value={option.value}>{option.key}</Option>) }
+              </Select>)}
+            </FormItem>
+            <FormItem label="Replicas Auto Balance" hasFeedback {...formItemLayoutForAdvanced}>
+              {getFieldDecorator('replicaAutoBalance', {
+                initialValue: 'ignored',
+              })(<Select>
+                <Option key={'ignored'} value={'ignored'}>ignored</Option>
+                <Option key={'disabled'} value={'disabled'}>disabled</Option>
+                <Option key={'least-effort'} value={'least-effort'}>least-effort</Option>
+                <Option key={'best-effort'} value={'best-effort'}>best-effort</Option>
+              </Select>)}
+            </FormItem>
+            <FormItem label="Allow Snapshot Removal During Trim" hasFeedback {...formItemLayoutForAdvanced}>
+              {getFieldDecorator('unmapMarkSnapChainRemoved', {
+                initialValue: 'ignored',
+              })(<Select>
+                <Option key={'enabled'} value={'enabled'}>Enabled</Option>
+                <Option key={'disabled'} value={'disabled'}>Disabled</Option>
+                <Option key={'ignored'} value={'ignored'}>Ignored (Remove Current Snapshot Chain during File System Trim)</Option>
```

Comment:
`Ignored (Remove Current Snapshot Chain during File System Trim)` is this message correct? I thought this is for `enabled`. 

We can just keep the display `ignored`.

Reply:
After discussing with David, we decide to remove the bracketed content.